manual/howtooperations/README.md

```diff
@@ -0,0 +1,6 @@
+** Operations How To Documents
+
+These are How-To docs for operations. They were originally created by Pete Johnson.
+
+The originals are [Google Docs](https://drive.google.com/drive/folders/192BgG6yxDbgtGKewFatUsar8kdAXT9Nb).
```

Comment:
Shouldn't the original documents be stored on the JMRI GitHub site?

Reply:
There's no native form for Google Docs.

Reply:
My apologies. I didn't thought about that.